understanding/22/findable-help.html

```diff
@@ -56,7 +56,7 @@ <h2>Intent of Findable Help</h2>
             </ul>
          <p>and is actively supported.</p>
       
-         <p>For a single page Web application, if the same URI routes to pages or views with different layouts or content, then each of those pages or views, have the supported ways of finding help in the same relative order</p>
+      <p>A <a>single page Web application</a> (compared to a <a>Web_Page</a>) shows multiple "pages" or views of content at the same URI. If a web application uses different URIs for each view of the content, that is considered multiple _web_pages_ because the URI changes.</p>
```

Comment:
although this has been merged already...wondering about those underscore `_` characters used. was that intentional?

Reply:
No, I corrected that prior to merging. (I think, checking... no, oops, updated in master.)